### DISCUSSION TITLE
WixBundleSourceProcessFolder in WiX 5

### DISCUSSION THREAD

**Discussion by MatthiasRInst** [wixtoolset/issues#8748] [Questions]
Hello,

Migrating our CustomBA to WiX 5, I stumbled across the notice that WixBundleSourceProcessFolder and WixBundleSourceProcessPath have been removed in WiX 5.

So is there a replacement or some recommended workaround to find the folder where the bundle is currently executed?

Best regards

Matthias Reuss

**Comment by gerhardmatzen** [comment]
I also just noticed that WixBundleSourceProcessPath is gone in WIX5. I had used it to restart elevated.
